R/pathwayEnrich.R

```diff
@@ -0,0 +1,131 @@
+
+# pathwayEnrich.R
+#'
+#' \code{PathwayEnrichment} a function that perform a pathway enrichment analysis 
+#' Return a barplot that show the enriched pathway and its p value(fisher test) that is smaller than 
+#' the ajusted p value. The null hypothesis for each fisher test: the pathway
+#' is not enriched in the given gene set.
+#'
+#' @param geneSet A list of gene symbol in the system that you want to perform pathway
+#'  enrichment analysis.
+#'  
+#' @param method method of cutoff that the visualization plot method can be BH or 
+#' bonferroni.
+#' reactInfo
+#' @param reactInfo a dataframe map the reactome pathway id to the reference gene symbol 
+#' 
+#' @return a barplot that show the enriched pathway and its p value(fisher test) 
+#' that is smaller than the ajusted p value.
+#' 
+#' @import ggplot2
```

Comment:
Shouldn't need import statements here, can be handled in the DESCRIPTION file.

Reply:
Thank you so much. I will be careful about that.

Reply:
Do you mean I should delete the import?

Reply:
I think imports are needed because the r check cannot pass without them.